domains/handikatriarlan.json

```
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "handikatriarlan"
+    },
+    "record": {
+        "CNAME": "cname.vercel-dns.com."
```

preview website gives 503 error
```suggestion
        "CNAME": "cname.vercel-dns.com"
```

---

I made it on purpose, because I'm still developing this website. but it will be completed in the near future.

---

> I made it on purpose, because I'm still developing this website. but it will be completed in the near future.

unfortunately we cant merge it unless its complete, as you can see in the pr requirements

---

> > I made it on purpose, because I'm still developing this website. but it will be completed in the near future.
> 
> unfortunately we cant merge it unless its complete, as you can see in the pr requirements

can you check again, please? i've finish and updated the website so no gives any error at all